src/main/java/seedu/address/model/room/UniqueRoomList.java

```diff
@@ -85,11 +87,13 @@ public void remove(Room toRemove) {
         if (!internalList.remove(toRemove)) {
             throw new RoomNotFoundException();
         }
+        FXCollections.sort(internalList);
```

Comment:
Probably no need to do this in `Remove` since removing from a sorted list keeps the list sorted

Reply:
Fixed